.github/workflows/release-build.yml

```diff
@@ -92,3 +92,14 @@ jobs:
           path: newrelic-php-agent/releases
           name: release-from-gha-${{ matrix.platform }}-${{ matrix.arch }}
           if-no-files-found: error
+
+      - name: Create release tags for Lambda and K8s Init Containers
+        uses: dev-hanz-ops/install-gh-cli-action@c78dbed4be2f8d6133a14a9a597ee12fd4ed5c93 # v3
+        with:
+          gh-cli-version: 2.63.2
```

Comment:
This is not needed - GitHub CLI is pre-installed and available on GitHub Actions runners:
* [Ubuntu 24.04](https://github.com/actions/runner-images/blob/ubuntu22/20240922.1/images/ubuntu/Ubuntu2404-Readme.md#cli-tools)
* [Ubuntu 22.04](https://github.com/actions/runner-images/blob/ubuntu22/20240922.1/images/ubuntu/Ubuntu2204-Readme.md#cli-tools)
* [Ubuntu 20.04](https://github.com/actions/runner-images/blob/ubuntu22/20240922.1/images/ubuntu/Ubuntu2004-Readme.md#cli-tools)